### DISCUSSION TITLE
If you don't know what to do next...

### DISCUSSION THREAD

**Discussion by Lineflyer** [cgeo/cgeo#9663] [General]
If it may happen anytime (but I doubt it ;) ) that you don't know what to do next while contributing to c:geo:

It might make sense to browse our issues for long existing feature requests or (even more relevant) long existing bugs. From time to time I try to scan those and sort out things which have been explicitly or implicitly solved or seem no longer really relevant due to negative or none feedback.

But still there are plenty of old things laying around:
- Bugs sorted by oldest first: https://github.com/cgeo/cgeo/issues?q=is%3Aissue+is%3Aopen+sort%3Acreated-asc+label%3ABug
- Feature requests sorted by oldest first: https://github.com/cgeo/cgeo/issues?q=is%3Aissue+is%3Aopen+sort%3Acreated-asc+label%3A%22Feature+Request%22

Feel free to take a look into those....be aware: Some of them might still need clarification whether its still needed to fix/implement.

**Comment by moving-bits** [comment]
I don't know if that will ever happen (not knowing what to do next in c:geo) :-), but thanks for the links anyway. May help to focus / prioritize things.
